app/templates/components/loading-indicator.hbs

```diff
@@ -0,0 +1,2 @@
+<span class="glyphicon glyphicon-refresh spinning" aria-hidden="true"></span>
```

Comment:
Is this span within the parent span (loading-indicator.js) necessary or could these attributes be moved to the outer span?

Reply:
I did it this way so that hbs code like:

```
{{#loading-indicator}}Generating Preview{{/loading-indicator}}
```

Would generate markup like:

```
<span class="loading-indicator">
  <span class="glyphicon ...><span>
  Generating Preview
</span>
```

The [boostrap examples](https://getbootstrap.com/docs/3.3/components/#glyphicons-examples) don't put any text inside of the `<span class="glyphicon"></span>`